app/globals.css

```
@@ -772,6 +806,190 @@
+    .noir-sticky-sidebar {
+      position: sticky;
+      top: 1rem;
+    }
+  }
+
+  /* Bold slanted thick marker style for titles/headings */
+  .font-marker {
+    font-family: var(--font-marker);
+    font-weight: 700;
+    transform: skewX(-4deg);
+    -webkit-text-stroke: 0.5px currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)